### PR TITLE
chore(docs): gradle plugin config

### PR DESCRIPTION
**Pull request by eshepelyuk**
References #1057 #1068

### CODE REVIEW

**Comment by raboof** [comment]
OK TO TEST

File: docs/src/main/paradox/buildtools/gradle.md [hunk 1]
```diff
@@ -4,37 +4,29 @@ To get started with Akka gRPC read the @ref[client](../client/index.md) or @ref[
 
 ## Configuring what to generate
 
-The plugin can be configured to generate either Java or Scala classes, and then server and or client for the chosen language.
-By default both client and server are generated and Java or Scala is selected depending on if the build
-has the `java` or `scala` plugin.
+This plugin is a wrapper for [protobuf-gradle-plugin](https://github.com/google/protobuf-gradle-plugin) and uses it for `.proto` files processing and code generation.
+Most of the settings could be configured using related setting of `protobuf-gradle-plugin` itself.
+Consult [protobuf-gradle-plugin](https://github.com/google/protobuf-gradle-plugin#protobuf-plugin-for-gradle-) documentation for details.
```
Comment (raboof) [inline docs/src/main/paradox/buildtools/gradle.md:9]: :+1:

File: docs/src/main/paradox/buildtools/gradle.md [hunk 2]
```diff
@@ -4,37 +4,29 @@ To get started with Akka gRPC read the @ref[client](../client/index.md) or @ref[
...
+
+The plugin can generate either `Java` or `Scala` classes, and then server and or client for the corresponding language.
```
Comment (raboof) [inline docs/src/main/paradox/buildtools/gradle.md:11]: I don't think we should format the words 'Java' and 'Scala' as code

File: docs/src/main/paradox/buildtools/gradle.md [hunk 3]
```diff
@@ -4,37 +4,29 @@ To get started with Akka gRPC read the @ref[client](../client/index.md) or @ref[
...
+By default both client and server are generated and `Java` or `Scala` is autodetected depending on the presence of source files with language extension in `src/main`.
```
Comment (raboof) [inline docs/src/main/paradox/buildtools/gradle.md:12]: :+1:

File: docs/src/main/paradox/buildtools/gradle.md [hunk 4]
```diff
@@ -43,30 +35,19 @@ Java
...
-      language = "Scala"
       generateClient = true
       generateServer = true
     }
     repositories {
-      mavenLocal()
+      // this repository is required only if SNAPSHOT version is used
+      maven {
+        url "https://dl.bintray.com/akka/snapshots"
+      }
```
Comment (raboof) [inline docs/src/main/paradox/buildtools/gradle.md:50]: Ah I didn't realize we had examples embedded in the docs like this - great to update them, thanks!

File: docs/src/main/paradox/buildtools/gradle.md [hunk 5]
```diff
@@ -87,23 +68,49 @@ To additionally generate server "power APIs" that have access to request metata,
     ```
     @@@
 
+## Protoc version
+
+Default version of `protoc` compiler is 3.4.0. 
+The version and the location of `protoc`can be changed using `protobuf-gradle-plugin` [settings](https://github.com/google/protobuf-gradle-plugin#locate-external-executables).
```
Comment (raboof) [inline docs/src/main/paradox/buildtools/gradle.md:74]:
```suggestion
The version and the location of `protoc` can be changed using `protobuf-gradle-plugin` [settings](https://github.com/google/protobuf-gradle-plugin#locate-external-executables).
```

File: docs/src/main/paradox/buildtools/gradle.md [hunk 6]
```diff
@@ -87,23 +68,49 @@ To additionally generate server "power APIs" that have access to request metata,
...
+      main = 'com.example.helloworld.GreeterServer'
+    }
+    ```
+    @@@
+
+Then, the server can then be started from the command line with:
+
+```
+./gradlew runServer
+```
```
Comment (raboof) [inline docs/src/main/paradox/buildtools/gradle.md:113]: :+1:

**Comment by eshepelyuk** [comment]
Updated, rebased.

**Pull request merged**